src/main/resources/anniv5Resources/localization/eng/runicpack/Keywordstrings.json

```
@@ -0,0 +1,12 @@
+[
+  {
+    "PROPER_NAME": "Runic",
+    "NAMES": ["runic"],
+    "DESCRIPTION": "At the start of the turn add a card which #yChannels orbs to your hand with #yEthereal and #yExhaust. When exited, gain #b1 #yFocus."
```

Should be a comma after "at the start of your turn". I'd also suggest "that" instead of "which" (both here and in the stance strings).
```suggestion
    "DESCRIPTION": "At the start of the turn, add a card that #yChannels orbs to your hand with #yEthereal and #yExhaust. When exited, gain #b1 #yFocus."
```